client/router_create.go

```diff
@@ -652,10 +652,14 @@ func (cli *VanClient) GetRouterSpecFromOpts(options types.SiteConfigSpec, siteId
 				controllerHosts = append(controllerHosts, controllerIngressHost)
 			}
 		}
+		siteServerSubject := types.TransportServiceName
+		if options.IsIngressNodePort() {
+			siteServerSubject = options.GetRouterIngressHost()
```

Comment:
Is this needed still (the ingress host should be added as a SAN)? The subject is limited to  less than 64 bytes as I recall.

Reply:
Nope, it was a leftover from another PR I have closed. I have just updated the PR.